### PR TITLE
Gabriel/home page rework

### PR DESCRIPTION
**Pull request by gdumouli**
- Changed the main page design
- Random event is displayed (and clickable) on main page

### CODE REVIEW

File: code/noRAM/app/src/main/java/com/example/noram/MainActivity.java [hunk 1]
```diff
@@ -192,4 +225,105 @@ private void updateAdminAccess(String uid) {
...
+                    endTime.format(DateTimeFormatter.ofPattern("MMM dd")),
+                    endTime.format(DateTimeFormatter.ofPattern("h:mm a"))
+            ));
+        }
+
+        // if event is currently happening, display "happening now!"
+        LocalDateTime current = LocalDateTime.now();
+        happeningNowText.setVisibility(startTime.isBefore(current) && endTime.isAfter(current) ? View.VISIBLE : View.GONE);
+
+        eventLocation.setText(event.getLocation());
```
Comment (WhoShotCupidd) [inline code/noRAM/app/src/main/java/com/example/noram/MainActivity.java:302]: could we get a few more comments here please

**Review by WhoShotCupidd — CHANGES_REQUESTED** [review]
Check the one comment I made please. 

Could we also make the event on the homepage a bit more centered the home page feels slightly segmented between the top and the bottom 

Also, did you look into the carousel design for the events?

**Comment by gdumouli** [comment]
> Check the one comment I made please.
> 
> Could we also make the event on the homepage a bit more centered the home page feels slightly segmented between the top and the bottom
> 
> Also, did you look into the carousel design for the events?

I don't think the carousel would be worthwhile: users don't stay on the main menu for very long and can use the event-list fragment if they want to scroll through available events. 
Was it for aesthetics that we wanted it?

**Comment by ColeDewis** [comment]
> > Check the one comment I made please.
> > Could we also make the event on the homepage a bit more centered the home page feels slightly segmented between the top and the bottom
> > Also, did you look into the carousel design for the events?
> 
> I don't think the carousel would be worthwhile: users don't stay on the main menu for very long and can use the event-list fragment if they want to scroll through available events. Was it for aesthetics that we wanted it?

not christiaan but yeah, its mostly for aesthetics. it's not really intended to be a way to browse events, more just as a gallery of "here's some stuff happening soon" that gives the home screen some flair

**Comment by WhoShotCupidd** [comment]
> > > Check the one comment I made please.
> > > Could we also make the event on the homepage a bit more centered the home page feels slightly segmented between the top and the bottom
> > > Also, did you look into the carousel design for the events?
> > 
> > 
> > I don't think the carousel would be worthwhile: users don't stay on the main menu for very long and can use the event-list fragment if they want to scroll through available events. Was it for aesthetics that we wanted it?
> 
> not christiaan but yeah, its mostly for aesthetics. it's not really intended to be a way to browse events, more just as a gallery of "here's some stuff happening soon" that gives the home screen some flair

yeah I think we should push this and we can look into the carousel later on

**Review by WhoShotCupidd — APPROVED** [review]
lgtm now

**Pull request merged**